app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,16 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str,
+                 musical_instrument: str,
+                 bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        return f"Elf ranger {self.nickname}. " \
+               f"{self.nickname} has bow of the {self._bow_level} level"
```

Comment:
```suggestion
        return (
            f"Elf ranger {self.nickname}. "
            f"{self.nickname} has bow of the {self._bow_level} level"
        )
```
Better to use brackets instead of \
Format the same in all places